### DISCUSSION TITLE
Windows 10/11 Set Minimum/Maximum Volume Limits

### DISCUSSION THREAD

**Discussion by Pheelix** [File-New-Project/EarTrumpet#1470] [Ideas]
Any chance you could add a setting to set minimum and maximum volume limits.  I know there is apps out there for setting maximum volume, but I am looking for something to set minimum volume.  With windows 10/11 if you have your balance set differently for left/right and then reduce your volume to 0 it will reset the balance.  If there is a way to do this or stop the balance reset when the volume is lowed to 0, please add it.

If it would be easier to add a way to set the balance using actions.  This way a action could be set on eartrumpet start to set the balance to L: X% / R: X%.  Or a action to set the balance for when volume is set from 0 to X it would set the balance.

**Comment by Aeolus702** [comment]
this can also be a accesability thing. my right ear needs to be louider, so what i do is turn down the left speaker to balance the audio.
